metaflow/plugins/kfp/kfp.py

```diff
@@ -393,28 +400,71 @@ def step_op_func(
     print("----")
 
     # TODO: Map username to KFP specific user/profile/namespace
+    # Note: we don't put this in a try catch block as below because
+    # Popen will not produce a error that will cause the Kubeflow step
+    # to stop
     with Popen(
         ["/bin/sh", "-c", cmd],
         stdin=PIPE,
         stdout=PIPE,
-        stderr=STDOUT,
+        stderr=PIPE,
```

Comment:
Why change this?
My understanding is that stderr with `stderr=STDOUT` is redirected to `stdout` and we only need to then capture `process.stdout`.  Am I correct in this assumption? It's worth testing this by printing to stderr and ensuring that we capture both stdout and stderr.  If I'm correct then we needn't capture into `stderr_buffer`.

Reply:
@talebzeghmi this is a good idea, but then I'd need to separate the `stderr_buffer` from the `stdout_buffer`, since `stdout` would contain both. I'd need to separate it out because I still need to upload two files. Otherwise, the command `python linear_flow_1.py --datastore=s3 logs kfp-79d69d38-4239-45d5-9cab-caf8a1142e9c/start/1` would fail because it's looking for both a `0.stdout.log` and `0.stderr.log`.

Reply:
I verified what you said is correct. In that case, `proncess.stdout` will contain both `stdout` and `stderr`. Thus, the file `0.stdout.log` will contain both `stdout` and `stderr`, and I'll upload an empty `0.stderr.log` to ensure the command `python linear_flow_1.py --datastore=s3 logs kfp-79d69d38-4239-45d5-9cab-caf8a1142e9c/start/1` works. Sound ok?